summary.md

```diff
@@ -0,0 +1,23 @@
+# Summary
+
+The Xing Framework is a combined web development environment for building both a server backend (in [Ruby on Rails](http://rubyonrails.org/)) and a single-page browser frontend (in [AngularJS](https://angularjs.org/)). 
+
+The Xing Framework provides integration and coordination of these two frameworks.  Specifically:
+
+## Features: Why you should use Xing for your next web application
+* **Development Support**. Tooling for building and launching the entire stack in development. With a single command, you can compile and boot your whole frontend and backend stack at once and open your app in a browser.
+* **API Language**.  A pre-defined, consistent, and hypermedia-enabled JSON API language.
+* **API Support**. Base classes in both AngularJS and Rails for consuming and producing hypermedia JSON resources in the Xing format.
```

Comment:
I might be more superlative here -- we don't just provide a base class... we have an entire library for producing and consuming an API